doctools/src/test/java/org/neo4j/doc/metatest/AsciiDocGeneratorTest.java

```diff
@@ -5,19 +5,19 @@
  * This file is part of Neo4j.
  *
  * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
+ * it under the terms of the GNU Affero General Public License as
```

Comment:
I don't think there is any reason to make the docs module AGPL, since it is no longer under the manual directory (which is AGPL since it depends on enterprise).